### PR TITLE
Refactor debug_marker

### PR DESCRIPTION
**Pull request by KeisukeShima**
## What kinds of PR

- [ ] New feature
- [x] Improve feature
- [ ] Bug fix

### CODE REVIEW

File: obstacle_stop_planner_refine/src/debug_marker.cpp [hunk 1]
```diff
@@ -314,249 +262,199 @@ visualization_msgs::msg::MarkerArray ObstacleStopPlannerDebugNode::makeVisualiza
...
+      visualization_msgs::msg::Marker::TEXT_VIEW_FACING,
+      autoware_utils::createMarkerScale(0.0, 0.0, 1.0),
+      autoware_utils::createMarkerColor(1.0, 1.0, 1.0, 0.999));
+
     marker.pose.position = *slow_down_obstacle_point_ptr_;
     marker.pose.position.z += 2.0;
     marker.pose.orientation.x = 0.0;
     marker.pose.orientation.y = 0.0;
     marker.pose.orientation.z = 0.0;
     marker.pose.orientation.w = 1.0;
```
Comment (kenji-miyake) [inline obstacle_stop_planner_refine/src/debug_marker.cpp]: Probably setting `orientation` can be removed?
There are some other lines which have the same problem.

Comment (KeisukeShima) [reply obstacle_stop_planner_refine/src/debug_marker.cpp]: Thanks for comment. I removed unnecessary code at 2a7c51a

**Review by kenji-miyake — DISMISSED** [review]
Cool!

**Comment by kenji-miyake** [comment]
Oh, but CI is failed... :pleading_face:

**Comment by KeisukeShima** [comment]
@kenji-miyake All tests has been fixed! Please review again :-)

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**